### PR TITLE
fix(filters): Not geo registered should be filtered by default(case)

### LINKED ISSUE #1219: Not geo registered should be filtered by default(case)
**I'm submitting a ...**  (check one with "x")

```
[x] bug report => search github for a similar issue or PR before submitting
```

**Current behavior** 

Not geo registered not filtered by default(case)
**Expected behavior** 

Not geo registered should be filtered by default(case)

### PR DESCRIPTION
**Pull request by yairtawil**
should resolve #1219

### CODE REVIEW

File: src/configuration/configuration.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 export const configuration = {
-	production: true
+	production: false
```
Comment (shaharido1) [inline src/configuration/configuration.ts]: need to change back to true?

File: src/app/packages/menu-items/filters/services/filters.service.ts [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
-			if (!currentFilter && !isNil(outerStateMetadata)) {
+			if (!currentFilter && Boolean(outerStateMetadata)) {
 				const [fieldName, metadata] = [filter.modelName, outerStateMetadata];
-				filters.push({ fieldName, metadata });
-			} else if (currentFilter && !isNil(outerStateMetadata)) {
+				filters.push({ fieldName, metadata, type: filter.type });
+			} else if (currentFilter && Boolean(outerStateMetadata)) {
 				currentFilter.metadata = outerStateMetadata;
-			} else if (currentFilter && isNil(outerStateMetadata)) {
+			} else if (currentFilter && Boolean(outerStateMetadata)) {
```
Comment (shaharido1) [inline src/app/packages/menu-items/filters/services/filters.service.ts]: !boolean

File: src/app/packages/menu-items/filters/effects/filters.effects.ts [hunk 3]
```diff
@@ -79,7 +79,8 @@ export class FiltersEffects {
 		const metaData: FilterMetadata =
 			this.genericTypeResolverService.resolveMultiInjection(FilterMetadata, resolveFilterFunction, false);
 
-		const currentFilterInit = facets.filters && facets.filters.find(({ fieldName }) => fieldName === filter.modelName);
+		const currentFilterInit = <any> facets.filters && facets.filters.find(({ fieldName }) => fieldName === filter.modelName);
```
Comment (shaharido1) [inline src/app/packages/menu-items/filters/effects/filters.effects.ts]: why any?

File: src/configuration/configuration.ts (hunk 1 above)
Comment (yairtawil) [reply src/configuration/configuration.ts]: Development should by default to be false..

Comment (shaharido1) [reply src/configuration/configuration.ts]: but it goes into production, isn't it?

**Pull request merged**